### PR TITLE
Update spawn.js

### PR DESCRIPTION
**Pull request by xgrommx**

### CODE REVIEW

**Comment by xgrommx** [comment]
Regarding with #952

**Comment by mattpodwysocki** [comment]
@xgrommx thanks

File: src/core/linq/observable/spawn.js [hunk 1]
```diff
@@ -40,9 +40,16 @@
           o.onCompleted();
           return;
         }
-        var value = toObservable.call(self, ret.value);
+        var obs = toObservable.call(self, ret.value);
+        var value = null;
+        var hasValue = false;
         if (Observable.isObservable(value)) {
```
Comment (mjolka) [inline src/core/linq/observable/spawn.js]: `value` will always be `null`. Did you mean `Observable.isObservable(obs)`?

**Comment by mattpodwysocki** [comment]
@mjolka @xgrommx yes, it was a bug and I have since fixed it

**Pull request merged**